build-tools/proprietary/proprietary.csproj

```diff
@@ -1,19 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Build">
```

Comment:
You can always remove `DefaultTargets="Build"`, that is the default in MSBuild if you leave this out.